### DISCUSSION TITLE
Please make tags searchable!

### DISCUSSION THREAD

**Discussion by seanyboi** [community/community#7542] [Other Feature Feedback, Questions, & Ideas] [answered]
Please, please, please make tags searchable. We use multiple different tags to release multiple different things from one repo and sometimes have to click so far back to find the last time we tagged something to make sure we iteratively name the next tag!

**Comment by MylesBorins** [comment, marked as answer]
At this time searchable tags will not be something that we are going to be pursuing. It would be a net new feature (could not build on top of existing search for releases)

**Reply by idanla-tr** [reply]
Our company still needs this feature. Is it in planning or still no?
